indexer/config/presets.go

```diff
@@ -25,6 +25,7 @@ var Presets = map[int]Preset{
 				L1CrossDomainMessengerProxy: common.HexToAddress("0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1"),
 				L1StandardBridgeProxy:       common.HexToAddress("0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"),
 				L1ERC721BridgeProxy:         common.HexToAddress("0x5a7749f83b81B301cAb5f48EB8516B986DAef23D"),
+				DisputeGameFactoryProxy:     common.HexToAddress("0x1111111111111111111111111111111111111111"),
```

Comment:
Uses all 1s because zero address is considered "no address" and indexer will throw on construction